clients/core-java/src/main/java/org/commonjava/indy/client/core/IndyClientHttp.java

```diff
@@ -317,10 +331,14 @@ public HttpResources getRaw( final String path, final Map<String, String> header
     {
         connect();
 
-        CloseableHttpResponse response = null;
+        CloseableHttpResponse response;
         try
         {
             final HttpGet req = newRawGet( buildUrl( baseUrl, path ) );
+            if ( headers != null )
```

Comment:
This is still a good idea to keep...but we'll have to think about a good way to expose it to the user.